app/vep/models/client_model.py

```diff
@@ -1 +1,39 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional
 
+def default_options():
```

Comment:
Do we need this ?

Reply:
yes, to add default options on line 17

Reply:
Right, We usually add constant variables.
https://stackoverflow.com/questions/77561280/const-pydantic-field-value

Reply:
Can we discuss this?